lib/moment-duration-format.js

```diff
@@ -1543,6 +1543,7 @@
 
     // init
     function init(context) {
+        context = context.default;
```

Comment:
I think it should be done at factory invocation spot https://github.com/jsmreese/moment-duration-format/blob/master/lib/moment-duration-format.js#L20 rather than in factory itself, and it's better to check `__esModule` flag and use `default` only in this case

Reply:
Hi osharper,
I've updated the code to get the default at the factory invocation. The flag cannot be checked there since its not line 20 but the requireJS anonymous invocation that was triggering. 
Anyways, this looks like a good way to go for us. Let me know if you're good.